src/utils/constants.tsx

```diff
@@ -0,0 +1,32 @@
+export const TABLE_COLUMN_NAMES = [
+  'Name',
+  'Sex',
+  'Born',
+  'Died',
+  'Mother',
+  'Father',
+];
+
+export const TABLE_SORT_FIELDS = [
+  'Name',
+  'Sex',
+  'Born',
+  'Died',
+];
+
+export const CENTURIES = [
+  '16',
+  '17',
+  '18',
+  '19',
+  '20',
+];
+
+export const SEARCH_FILTER = {
+  All: null,
+  Male: 'm',
+  Female: 'f',
```

Comment:
This search filter reffers only to sex, you can be more specific with name of constant